go/ct/st/state.go

```diff
@@ -46,6 +48,49 @@ const (
 	NumStatusCodes                   // not an actual status
 )
 
+type SelfDestructEntry struct {
+	account    vm.Address
+	oldBalance vm.Value
+}
+
+func NewSelfDestructEntry(acc vm.Address, bal vm.Value) SelfDestructEntry {
+	return SelfDestructEntry{account: acc, oldBalance: bal}
+}
+
+func (s SelfDestructEntry) String() string {
+	return fmt.Sprintf("account: %v, oldBalance: %v", s.account, s.oldBalance)
+}
+
+func (s SelfDestructEntry) MarshalJSON() ([]byte, error) {
+	return json.Marshal(s.String())
```

Comment:
nit: it kinda breaks the JSON encoding if you encode two nested elements in a string that needs to be parsed again. It might be cleaner to create a
```
type serializableEntry struct {
   Account vm.Address
   Beneficiary vm.Address
}
```
copy all data into that, and use the JSON serialization on that object. Wdyt?